### ISSUE TITLE
Mismatch in the result (AP)

### ISSUE THREAD

**Issue by jayaramreddy10** [rhett-chen/graspness_implementation#38]
Hi, 

I have a query on test_seen result. When I evaluate kinect checkpoint on test_seen dataset, I get much lesser AP compared to what's reported. Could you please let me know if I am executing it wrong?

Command: CUDA_VISIBLE_DEVICES=0 python test.py --camera kinect --dump_dir logs/logs_kn --checkpoint_path ckpts/minkuresunet_kinect.tar --batch_size 1 --dataset_root data --infer --eval --collision_thresh -1

Result: 
![Screenshot from 2024-07-01 17-50-52](https://github.com/rhett-chen/graspness_implementation/assets/111297422/820f9d4e-5171-460a-8b76-7893aeb4f122)

Also, I was using the pretrained weights from this github (https://github.com/graspnet/graspness_unofficial). If these are not the correct ones, could you please share the correct files to mailid: ramreddyai1010@gmail.com.

**Comment by wanghq-thu** [comment]
> Hi,
> 
> I have a query on test_seen result. When I evaluate kinect checkpoint on test_seen dataset, I get much lesser AP compared to what's reported. Could you please let me know if I am executing it wrong?
> 
> Command: CUDA_VISIBLE_DEVICES=0 python test.py --camera kinect --dump_dir logs/logs_kn --checkpoint_path ckpts/minkuresunet_kinect.tar --batch_size 1 --dataset_root data --infer --eval --collision_thresh -1
> 
> Result: ![Screenshot from 2024-07-01 17-50-52](https://private-user-images.githubusercontent.com/111297422/344673298-820f9d4e-5171-460a-8b76-7893aeb4f122.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEyOTA4MTksIm5iZiI6MTcyMTI5MDUxOSwicGF0aCI6Ii8xMTEyOTc0MjIvMzQ0NjczMjk4LTgyMGY5ZDRlLTUxNzEtNDYwYS04Yjc2LTc4OTNhZWI0ZjEyMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxOFQwODE1MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iOTY0ODhiZDBlNDU3MTAxZjg2YmQyM2QzNWEyOGM2YzAwNGZmMDJhZDY5NTQwNDk1ZmRkZTliMzM4YmM0ODhiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ydbuZrsQMOY9JizzucvCUpCU4LDY6Jq52fkzDfetDmw)
> 
> Also, I was using the pretrained weights from this github (https://github.com/graspnet/graspness_unofficial). If these are not the correct ones, could you please share the correct files to mailid: [ramreddyai1010@gmail.com](mailto:ramreddyai1010@gmail.com).

Hello, I have also encountered a similar situation. The accuracy is only a little over 10 points. Have you made any progress now. Have you trained the model yourself.

**Comment by jayaramreddy10** [comment]
Hey, I tried with the checkpoint from this repo. (https://github.com/user432/gamma). And it gave me AP of close to 0.55 to 0.60.
